inst/examples/build-a-box/deploy.R

```diff
@@ -1,3 +1,6 @@
+withr::local_temp_libpaths()
+install.packages("shiny")
```

Comment:
`local_temp_libpaths()`:

> Temporarily prepend a new temporary directory to the library paths.

The goal is to install `shiny` in front of the rest of the packages used in CI and only for the deployment of this app.